docs/src/pages/[platform]/theming/default-theme/colors.tsx

```diff
@@ -1,33 +0,0 @@
-import { Flex, defaultTheme, View } from '@aws-amplify/ui-react';
-
-const { tokens } = defaultTheme;
-
-const baseColors = [
-  'red',
-  'orange',
-  'yellow',
-  'green',
-  'teal',
-  'blue',
-  'purple',
-  'pink',
-];
-
-export const Colors = () => {
```

Comment:
Looks like someone started building out the default-theme and just never put it on a page?

Reply:
yea, looks like it's not used anywhere 😂